### PR TITLE
doc: Use helm repo instead of OCI image as the installation source

### PR DESCRIPTION
**Pull request by CH3CHO**
### Ⅰ. Describe what this PR did

Use helm repo instead of OCI image as the installation source

### Ⅴ. Special notes for reviews

### CODE REVIEW

File: README_EN.md [hunk 1]
```diff
@@ -127,8 +127,8 @@ kubectl.exe config use-context kind-higress
 #### step 3. install higress
 
 ```bash
-kubectl create ns higress-system
-helm install higress -n higress-system oci://higress-registry.cn-hangzhou.cr.aliyuncs.com/charts/higress-local
+helm repo add higress.io https://higress.io/helm-charts
+helm install higress higress.io/higress-local -n higress-system --create-namespace
```
[the remark this answers is not in the document]
Comment (CH3CHO) [reply README_EN.md:131]: Do we need to change the helm chart to achieve this?

[the remark this answers is not in the document]
Comment (johnlanni) [reply README_EN.md:131]: There is already a `global.kind` option that can do this.

[the remark this answers is not in the document]
Comment (CH3CHO) [reply README_EN.md]: Done.

**Review by johnlanni — APPROVED** [review]
LGTM

**Pull request merged**